perlio.h

```diff
@@ -63,6 +63,16 @@ typedef PerlIOl *PerlIO;
 #define PerlIO PerlIO
 #define PERLIO_LAYERS 1
 
+/*
+=for apidoc_section $io
+=for apidoc Amu||PERLIO_FUNCS_DECL|PerlIO * func
+Declare C<func> to be a PerlIO function, that is, of type C<PerlIO_funcs>.
```

Comment:
Declare C<func> to be a PerlIO function table, that is, of type C<PerlIO_funcs>.